docs/docs-content/integrations/aws-autoscaler.md

```diff
@@ -425,8 +816,8 @@ data "spectrocloud_registry" "public_registry" {
 
 data "spectrocloud_pack_simple" "aws-cluster-autoscaler" {
  name    = "aws-cluster-autoscaler"
- version = "1.26.3"
- type = "helm"
+ version = "1.28.2"
+ type = "spectro"
```

Comment:
```suggestion
 type = "helm"
```

`spectro` is not a supported value ([more info here].(https://registry.terraform.io/providers/spectrocloud/spectrocloud/latest/docs/data-sources/pack_simple)).

How do I know it's a Helm type? Because of the `pack.json`

https://github.com/spectrocloud/pax/blob/master/stable/addon/systemapps/aws-cluster-autoscaler-1.29.2/pack.json#L10-L12

Reply:
Oh, nice. :)
The spectro value came from the API payload, I thought it was valid :sweat_smile: Thank you!